scripts/pre_commit_linter.py

```diff
@@ -979,6 +984,26 @@ def _check_comments(all_files):
                         print '%s --> Line %s: %s' % (
                             filename, line_num + 1, message)
 
+                # Check that comment starts with a space.
+                if space_regex.match(line):
+                    message = (
+                        'There should be a space at the beginning '
+                        'of the comment.')
+                    failed = True
+                    print '%s --> Line %s: %s' % (
+                        filename, line_num + 1, message)
+
+                if not line.startswith('# coding:') and not (
```

Comment:
There can be cases like [here](https://github.com/oppia/oppia/blob/develop/appengine_config.py#L76) and [here](https://github.com/oppia/oppia/blob/develop/core/domain/state_domain.py#L770) where the comments should begin with a small letter. Look for similar cases and try to find a solution.

Reply:
Hi @seanlip, some comments can begin with small letter like on the above examples, they are especially keywords. Should those keywords be enclosed between something(maybe?) to avoid confusion and distinguish them and comments can begin with capital or a quote(or whatever we are enclosing it between)?

Reply:
I think these are manually fixable. For the first one we can replace oppia_tools/ with "The oppia_tools/ folder". For the second, we can remove the typeinfo from the inline comments; it's out of line with other similar classes.

Reply:
@seanlip In my captial checking regex it's only checking for full words, so things like function names with "_" and "." in them are excluded. Would this be a viable strategy or should I go ahead and manually change all these commented function names like you asked?

Reply:
I think that sounds fine as well!

Reply:
Fixed ALL pylint capitalization errors in our codebase that failed Travis CI by hand.